src/js/jobs/sagas.js

```diff
@@ -47,5 +75,8 @@ export function* watchJobs() {
     yield takeEvery(CANCEL_JOB.REQUESTED, cancelJob);
     yield takeEvery(ARCHIVE_JOB.REQUESTED, archiveJob);
     yield takeLatest(WS_UPDATE_JOB, wsUpdateJob);
+    yield takeLatest(WS_INSERT_JOB, wsInsertJob);
+    yield takeLatest(WS_REMOVE_JOB, wsRemoveJob);
```

Comment:
```suggestion
    yield takeLatest([WS_INSERT_JOB, WS_REMOVE_JOB], getJobCount);
```

Reply:
Useful bit of syntax, will update to this and remove the wsInsertJob and wsRemoveJob functions.

Reply:
Updated